Data/models/DataController.swift

```diff
@@ -235,49 +237,24 @@ public class DataController {
         return DataController.sharedInMemory.container.viewContext
     }
     
-    static func save(context: NSManagedObjectContext?) {
```

Comment:
Do we not need the "save" method anywhere ? Did we remove its usage before ?

Reply:
we do not, you are supposed to save with `perform` method, this was method was not used